frontend/src/app/pages/exercises/exercise/shared/core/drag-element.service.ts

```diff
@@ -179,83 +209,128 @@ export class DragElementService {
-                const height = SimulatedRegion.image.height / 23.5;
-                const width = height * SimulatedRegion.image.aspectRatio;
-                this.exerciseService.proposeAction(
-                    {
-                        type: '[SimulatedRegion] Add simulated region',
-                        simulatedRegion: SimulatedRegion.create(
-                            {
-                                x: position.x - width / 2,
-                                y: position.y + height / 2,
-                            },
-                            {
-                                height,
-                                width,
-                            },
-                            'Einsatzabschnitt ???'
-                        ),
-                    },
-                    true
-                );
+            case 'simulatedRegion':
+                {
+                    // This ratio has been determined by trial and error
+                    const height = SimulatedRegion.image.height / 23.5;
+                    const width = height * SimulatedRegion.image.aspectRatio;
+                    const simulatedRegion = SimulatedRegion.create(
+                        {
+                            x: position.x - width / 2,
+                            y: position.y + height / 2,
+                        },
+                        {
+                            height,
+                            width,
+                        },
+                        'Einsatzabschnitt ???'
+                    );
+                    this.exerciseService.proposeAction(
+                        {
+                            type: '[SimulatedRegion] Add simulated region',
+                            simulatedRegion,
+                        },
+                        true
+                    );
+                    createdElement = simulatedRegion;
+                }
                 break;
-            }
+
             default:
                 break;
         }
+
+        this.executeDropSideEffects(pixel, createdElement);
     };
 
+    private executeDropSideEffects(pixel: Pixel, createdElement: Elem | null) {
+        if (
+            createdElement === null ||
+            !this.olMap ||
+            !this.layerFeatureManagerDictionary
+        ) {
+            return;
+        }
+        this.olMap.forEachFeatureAtPixel(pixel, (droppedOnFeature, layer) => {
+            // Skip layer when unset
+            if (layer === null || !this.layerFeatureManagerDictionary) {
+                return;
+            }
+            // We stop propagating the event as soon as the onFeatureDropped function returns true
+            return this.layerFeatureManagerDictionary
+                .get(layer as VectorLayer<VectorSource>)!
+                .onFeatureDrop(
+                    createdElement as Elem,
+                    droppedOnFeature as Feature
+                );
+        });
```

Comment:
Maybe you can move this code to a function out of this class to share it with `OlMapInteractionsManager.handleTranslateEnd`

Reply:
I thought about that, but while they are similar in Structure, they are different in break conditions and the Way the onFeatureDrop Function is called, so I suggest leaving it as is.